innovation_sweet_spots/getters/crunchbase.py

```diff
@@ -0,0 +1,33 @@
+"""
+innovation_sweet_spots.getters.crunchbase
+
+Module for easy access to downloaded CB data
+
+"""
+import pandas as pd
+from innovation_sweet_spots.getters.path_utils import CB_PATH
+
+
+def get_crunchbase_category_groups():
+    return pd.read_csv(f"{CB_PATH}/crunchbase_category_groups.csv")
```

Comment:
The paths in `crunchbase.py` and `gtr.py` are pathlib objects so they don't need f-strings. For example, `f"{CB_PATH}/crunchbase_category_groups.csv"` can be written `CB_PATH / "crunchbase_category_groups.csv"`.

Reply:
Good point, thanks!